packages/core/types/index.js

```diff
@@ -286,8 +287,12 @@ export interface Dependency {
   +target: ?Target;
   +sourceAssetId: ?string;
   +sourcePath: ?string;
-  +symbols: Map<Symbol, Symbol>;
+  // (imported symbol -> variable that it is used as)
+  +symbols: $ReadOnlyMap<Symbol, Symbol>;
+  +symbolsLocs: $ReadOnlyMap<Symbol, SourceLocation>;
```

Comment:
Same idea here: Map<Symbol, {identifier: Symbol, loc: SourceLocation>, and make internal?